src/cloud/ops.rs

```diff
@@ -143,6 +166,18 @@ pub async fn create_cloud_instance(
     Ok(())
 }
 
+pub async fn upgrade_cloud_instance(
+    client: &CloudClient,
+    instance: &CloudInstanceUpgrade,
+) -> anyhow::Result<()> {
+    let url = format!("orgs/{}/instances/{}", instance.org, instance.name);
+    let instance: CloudInstance = client
+        .put(url, serde_json::to_value(instance)?)
+        .await?;
+    wait_instance_upgrade(instance, client, false).await?;
```

Comment:
`quiet` is never `true` as far as I can see. Is some instance propagation option missing, or why there is a flag?

Reply:
I'm not sure why it was added originally. Neither `create` nor `upgrade` command supports that flag, so I removed it